bats/fb-install-katello.bats

```diff
@@ -93,6 +93,7 @@ setup() {
 }
 
 @test "install CLI (hammer)" {
+  yum clean all
```

Comment:
Yeah, the problem here is that the yum metadata is retrieved at the start of the process, and the installation may take ~30 minutes.  So you have a 30 minute window where if foreman re-deploys their nightly repos any foreman package install may break.  I think this is the only place where that happens?